src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -939,6 +939,10 @@ static Library()
                     returnBehavior: ReturnBehavior.ReturnFalseIfAnyArgIsBlank,
                     targetFunction: IsToday)
             },
+            {
+                BuiltinFunctionsCore.Language,
+                NoErrorHandling(Language)
```

Comment:
>NoErrorHandling [](http://example.com/codeflow?start=16&length=15)

Why No Error Handling?

Reply:
It has `returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank` which may be useful ;)

Reply:
This is language function, not in scope of this PR. but this is what Calos suggested.

Reply:
Oops my bad thought it was PlainText :D

Reply:
Thanks